salt/utils/vt.py

```diff
@@ -788,38 +666,10 @@ def getwinsize(self):
                     "Can't check the size of the terminal since we're not "
                     "connected to the child process."
                 )
+            return getwinsize(self.child_fd)
 
-            TIOCGWINSZ = getattr(termios, "TIOCGWINSZ", 1074295912)
-            packed = struct.pack(b"HHHH", 0, 0, 0, 0)
-            ioctl = fcntl.ioctl(self.child_fd, TIOCGWINSZ, packed)
-            return struct.unpack(b"HHHH", ioctl)[0:2]
-
-        def setwinsize(self, rows, cols):
-            """
-            This sets the terminal window size of the child tty. This will
-            cause a SIGWINCH signal to be sent to the child. This does not
-            change the physical window size. It changes the size reported to
-            TTY-aware applications like vi or curses -- applications that
-            respond to the SIGWINCH signal.
-
-            Thank you for the shortcut PEXPECT
-            """
-            # Check for buggy platforms. Some Python versions on some platforms
-            # (notably OSF1 Alpha and RedHat 7.1) truncate the value for
-            # termios.TIOCSWINSZ. It is not clear why this happens.
-            # These platforms don't seem to handle the signed int very well;
-            # yet other platforms like OpenBSD have a large negative value for
-            # TIOCSWINSZ and they don't have a truncate problem.
-            # Newer versions of Linux have totally different values for
-            # TIOCSWINSZ.
-            # Note that this fix is a hack.
-            TIOCSWINSZ = getattr(termios, "TIOCSWINSZ", -2146929561)
-            if TIOCSWINSZ == 2148037735:
-                # Same bits, but with sign.
-                TIOCSWINSZ = -2146929561
-            # Note, assume ws_xpixel and ws_ypixel are zero.
-            packed = struct.pack(b"HHHH", rows, cols, 0, 0)
-            fcntl.ioctl(self.child_fd, TIOCSWINSZ, packed)
+        def setwinsize(self, child, rows=80, cols=80):
+            setwinsize(self.child_fd, rows, cols)
```

Comment:
There's no scoping issues with the functions being named the same?

Reply:
No, it's a simple closure. :)